memgraph_versioned_docs/version-2.3.1/connect-to-memgraph/drivers/php.md

```diff
@@ -37,47 +37,46 @@ following code to it:
 
 ```php
 <?php
-
 require_once __DIR__ . '/vendor/autoload.php';
 
-// Create new Bolt instance
-$bolt = new \Bolt\Bolt(new \Bolt\connection\StreamSocket());
-// Set Bolt protocol version if needed
-$bolt->setProtocolVersions(4.0);
-// Connect to database
-$bolt->init('MyClient/1.0', '', '');
-
-// Execute query
-$res = $bolt->run(
+// Create connection class and specify target host and port
+$conn = new \Bolt\connection\Socket();
+// Create new Bolt instance and provide connection object
+$bolt = new \Bolt\Bolt($conn);
+// Build and get protocol version instance which creates connection and executes handshake
+$protocol = $bolt->build();
+// Login to database with credentials
+$protocol->hello(\Bolt\helpers\Auth::basic('username', 'password'));
+// Execute query with parameters
+$stats = $protocol->run(
     'CREATE (a:Greeting) SET a.message = $message RETURN id(a) AS nodeId, a.message AS message',
     ['message' => 'Hello, World!']
 );
-// Pull records from last query
-$rows = $bolt->pull();
-
+// Pull records from last executed query
+$rows = $protocol->pull();
 echo 'Node ' . $rows[0][0] . ' says: ' . $rows[0][1];
 ```
 
-**3.** Create a `composer.json` file with the following contents:
-
-```json
-{
-  "name": "memgraph/myapp",
-  "description": "This is a simple Hello World app.",
-  "require": {
-    "php": ">=7.1",
-    "stefanak-michal/bolt": "^2.7.2"
-  }
-}
+If you need SSL connection you have to replace `Socket` instance with `StreamSocket` and enable SSL with additional method.
+
+```php
+$conn = new \Bolt\connection\StreamSocket('URI or IP', 7687);
+$conn->setSslContextOptions([
+    'verify_peer' => true
+]);
 ```
```

Comment:
```suggestion
```nocopy
$conn = new \Bolt\connection\StreamSocket('URI or IP', 7687);
$conn->setSslContextOptions([
    'verify_peer' => true
]);
```
```
Every copyable code block should be a working code. Therefore, I think that this block should have `nocopy` signature. In the next version of this docs site (after we merge this), we'll add examples of working code with and without SSL.